### PR TITLE
amp-bind: Fix more integration tests

### PR DESCRIPTION
**Pull request by choumx**
Fixes all tests on Saucelabs (except for one). Partial for #9571. 

- Add missing polyfills (e.g. `Array.include`) to web worker.
- Remove use of polyfilled functions in top-level scope of modules.*
    - `map()` in bind-validator.js
    - `Array.includes` in bind-expression.js
- Add fallback for `Event` constructor.
- Add `ownProperty()` function to utils/object.js.

*May also be fixed by importing `web-worker-polyfills`, but I thought that looked weird.

### CODE REVIEW

File: test/integration/test-amp-bind.js [hunk 1]
```diff
@@ -19,14 +19,15 @@ import {batchedXhrFor, bindForDoc} from '../../src/services';
...
-// TODO(choumx): Unskip once #9571 is fixed.
-describe.skip('amp-bind', function() {
+describe.configure().retryOnSaucelabs().run('amp-bind', function() {
   let fixture;
   let ampdoc;
   let sandbox;
   let numSetStates;
   let numMutated;
 
+  this.timeout(5000);
```
Comment (rsimha-amp) [inline test/integration/test-amp-bind.js]: Maybe add a comment explaining why this test needs 5s?

**Comment by rsimha-amp** [comment]
FYI, failing travis build is due to https://www.traviscistatus.com/incidents/bnt2wtxpgs39

File: test/integration/test-amp-bind.js (hunk 1 above)
Comment (choumx) [inline test/integration/test-amp-bind.js]: This reverts #9558 since this PR fixes the real root cause of the timeouts.

**Pull request merged**